src/com/facebook/buck/jvm/java/intellij/ij-module.st

```diff
@@ -49,7 +49,14 @@
         %if(generatedFolder.isAndroidResources)% type="java-resource" %endif%/>
     </content>
 }%
+
+%if(!androidFacet.enabled && java_sdk)%
+    <orderEntry type="jdk" jdkName="%java_sdk%" jdkType="JavaSDK" />
```

Comment:
I had a bunch of code that would attempt to read IntelliJ's config and determine whether these SDKs are actually defined (with the intent to figure out if the .jar files they include would be the ones buck would use during the build). IIRC that was like 80% complete, would you be interested in taking that over?

Assuming that the developers will have the correct SDK set up has not worked well for us internally. Any changes to the default are painful and it takes a long time for everyone to update their IJ setup.